acceptance/tests/custom_facts/using_win32ole_should_not_hang.rb

```diff
@@ -0,0 +1,29 @@
+test_name 'C92060: Custom facts should not hang Facter when using win32ole' do

```

Comment:
Ah, so the original file that I moved and updated contains carriage returns which is causing you issues. I'll remove them so all the tests are consistent.